.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   - /tmp/vscode-ripgrep-cache-1.5.7
   - dev-packages/application-manager/node_modules
   - dev-packages/application-package/node_modules
+  - dev-packages/cli/node_modules
```

Comment:
did you change it manually or `scripts/prepare-travis` insert it? We should not touch this section. Wonder how we missed it before.

Reply:
It was automatically updated somehow, likely through the script you mentioned.

Reply:
Ok, weird that it was not there before